crates/solidity/testing/snapshots/cst_output/InterfaceDefinition/sample_counter/generated/0.4.11.snap

```diff
@@ -1,6 +1,44 @@
 # This file is generated automatically by infrastructure scripts. Please don't edit by hand.
 
-Errors: 0
+Error: subtrahend matches minuend
```

Comment:
If I'm reading this correctly, this is a bug in parsing the `Identifier` production. Is this error message correct? Shouldn't the `minuend` be parsed/returned successfully in this case?

Reply:
I need to investigate this, because it is correct parsing, and a sign that `difference` is working, but it shouldn't manifest.

Reply:
Fixed